core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/parser/antlr/DataRepoEvaluateToAttributeVisitor.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.rawls.expressions.parser.antlr
+
+import org.broadinstitute.dsde.rawls.expressions.Transformers._
+import org.broadinstitute.dsde.rawls.expressions.parser.antlr.TerraExpressionParser.EntityLookupContext
+
+import scala.collection.JavaConverters._
+
+case class ParsedDataRepoExpression(relationships: List[String], columnName: String, expression: LookupExpression, tableAlias: String) {
+  val qualifiedColumnName = s"$tableAlias.$columnName"
+}
+
+class DataRepoEvaluateToAttributeVisitor(rootTableAlias: String) extends TerraExpressionBaseVisitor[Seq[ParsedDataRepoExpression]] {
+  override def defaultResult(): Seq[ParsedDataRepoExpression] = Seq.empty
+
+  override def aggregateResult(aggregate: Seq[ParsedDataRepoExpression], nextResult: Seq[ParsedDataRepoExpression]): Seq[ParsedDataRepoExpression] = {
+    aggregate ++ nextResult
+  }
+
+  override def visitEntityLookup(ctx: EntityLookupContext): Seq[ParsedDataRepoExpression] = {
+    val relations = ctx.relation().asScala.toList
+
+    val tableAlias = if (relations.isEmpty) {
+      rootTableAlias
+    } else {
+      relations.last.getText
```

Comment:
first, this is untested until we get relationships in.

The purpose of this is to give the table in the BQ query and alias so it can be referred to unambiguously in the select statement.